src/map/battle.cpp

```diff
@@ -5601,22 +5601,26 @@ static int battle_calc_attack_skill_ratio(struct Damage* wd, struct block_list *
 			break;
 		case WH_HAWKRUSH:
 			skillratio += -100 + 500 * skill_lv + 5 * sstatus->con;
+			if (sd)
+				skillratio += skillratio * pc_checkskill(sd, WH_NATUREFRIENDLY) / 10;
```

Comment:
Unrelated according to patch notes?

Reply:
Yes, this is an additional fix. Other fixes will be in another PR from now on